### PR TITLE
[hi] Localizing debugging.md

### LINKED ISSUE #1067: [New term] `Debugging` for `Hindi`
#### What this term is about? (briefly)
- Debugging is the process of finding & resolving bugs(errors) from a computer program, software, or system to get the desired result.
#### Why this term needs to be added? (briefly) 
- Localizing this term in `Hindi` will help `Hindi` folks learn about this term in their native language.

Link here:- https://glossary.cncf.io/debugging/

### PR DESCRIPTION
**Pull request by Omkar0114**
fixes #1067

### CODE REVIEW

File: content/hi/debugging.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+---
+title: डिबगिंग(Debugging)
+status: Feedback Appreciated
```
Comment (Garima-Negi) [inline content/hi/debugging.md:3]: please check: [https://github.com/cncf/glossary/blob/main/content/en/debugging.md](https://github.com/cncf/glossary/blob/main/content/en/debugging.md)

the status seems to have changed. Make sure your main branch is latest y running 'git pull'. you may then rebase this branch with main

**Review by Garima-Negi — CHANGES_REQUESTED** [review]

**Comment by Garima-Negi** [comment]
Hi, please retitle the PR to include [hi] as well as specify that you are localizing a term , not creating it.

File: content/hi/debugging.md (hunk 1 above)
Comment (Omkar0114) [reply content/hi/debugging.md]: I changed status. You can review it. @Garima-Negi

**Comment by Omkar0114** [comment]
I made all the requested changes. Take a look at it. @Garima-Negi @anubha-v-ardhan :)

**Comment by Omkar0114** [comment]
Hey, @Garima-Negi I made all the requested changes, Can you please review them again?

File: content/hi/debugging.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+## समस्या
+
+सॉफ्टवेयर विकास एक जटिल गतिविधि है जो बग को पेश किए बिना कोड लिखना लगभग असंभव बना देती है। वे बग कोड की ओर ले जाते हैं जो निष्पादित होने पर वांछित (अपरिभाषित व्यवहार) 
+के रूप में कार्य नहीं करेंगे। कोई एप्लिकेशन कितना महत्वपूर्ण है, इस पर निर्भर करते हुए, बग का महत्वपूर्ण नकारात्मक प्रभाव हो सकता है - आर्थिक रूप से और मानव जीवन पर भी। 
+आमतौर पर, एप्लिकेशन कोड को विभिन्न चरणों या वातावरण से गुजरना पड़ता है जहां इसका परीक्षण किया जाता है। एक आवेदन जितना अधिक महत्वपूर्ण होगा, परीक्षण उतना ही सटीक होना चाहिए।
+
+## समाधान
+
+जब बग(त्रुटीया) दिखाई देते हैं, तो इंजीनियरों को उत्पादन प्रणालियों के लिए अवांछित व्यवहार को कम करने के लिए आवेदन को डिबग (जैसे, ढूंढना और ठीक करना) करना पड़ता है।
```
[the remark this answers is not in the document]
Comment (ShivamTyagi12345) [reply content/hi/debugging.md:20]: इंजीनियरों can be just used in its singular form

File: content/hi/debugging.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+के रूप में कार्य नहीं करेंगे। कोई एप्लिकेशन कितना महत्वपूर्ण है, इस पर निर्भर करते हुए, बग का महत्वपूर्ण नकारात्मक प्रभाव हो सकता है - आर्थिक रूप से और मानव जीवन पर भी। 
+आमतौर पर, एप्लिकेशन कोड को विभिन्न चरणों या वातावरण से गुजरना पड़ता है जहां इसका परीक्षण किया जाता है। एक आवेदन जितना अधिक महत्वपूर्ण होगा, परीक्षण उतना ही सटीक होना चाहिए।
+
+## समाधान
+
+जब बग(त्रुटीया) दिखाई देते हैं, तो इंजीनियरों को उत्पादन प्रणालियों के लिए अवांछित व्यवहार को कम करने के लिए आवेदन को डिबग (जैसे, ढूंढना और ठीक करना) करना पड़ता है। 
+डिबगिंग कोई आसान काम नहीं है क्योंकि इंजीनियरों को अवांछित व्यवहार के स्रोत को ट्रैक करना होता है। इसे कोड के बारे में ज्ञान और रनटाइम पर निष्पादन संदर्भ की आवश्यकता होती है। 
+यह वह जगह है जहाँ विभिन्न डिबगिंग तकनीक और उपकरण काम में आते हैं। उदाहरण के लिए, लॉग, ट्रेस और मेट्रिक्स का विश्लेषण, उत्पादन में सीधे डिबगिंग के लिए उपयोग किया जाता है।
+संबंधित निष्पादन संदर्भ का विश्लेषण करते समय डेवलपर्स रनटाइम पर कोड के माध्यम से कदम उठाने के लिए इंटरैक्टिव डिबगिंग का उपयोग कर सकते हैं। एक बार जब वे विफलता के स्रोत की पहचान कर लेते हैं, 
+तो वे कोड को सही करते हैं और बग फिक्स या पैबंद(पैच) बनाते हैं।
```
Comment (ShivamTyagi12345) [inline content/hi/debugging.md:24]: बग फिक्स या पैबंद(पैच) can be chopped down to पैच

**Review by ShivamTyagi12345 — CHANGES_REQUESTED** [review]

**Comment by Omkar0114** [comment]
Hey, @ShivamTyagi12345 I made all the requested changes. You can review it again :)

File: content/hi/debugging.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+---
+
+## यह क्या है
+
+डिबगिंग यह एक वांछित परिणाम प्राप्त करने के लिए कंप्यूटर प्रोग्राम, सॉफ़्टवेयर या सिस्टम से बग को खोजने और हल करने की प्रक्रिया है। 
+बग एक दोष या समस्या है जिसके कारण गलत या अप्रत्याशित परिणाम मिलते हैं।
+
+## समस्या
+
+सॉफ्टवेयर विकास एक जटिल गतिविधि है जो बग को पेश किए बिना कोड लिखना लगभग असंभव बना देती है। वे बग कोड की ओर ले जाते हैं जो निष्पादित होने पर वांछित (अपरिभाषित व्यवहार)
```
Comment (Garima-Negi) [inline content/hi/debugging.md:14]: let's rephrase this line better: वे बग कोड की ओर ले जाते हैं जो निष्पादित होने पर वांछित (अपरिभाषित व्यवहार) के रूप में कार्य नहीं करेंगे।

**Review by Garima-Negi — CHANGES_REQUESTED** [review]

File: content/hi/debugging.md (hunk 4 above)
Comment (Omkar0114) [reply content/hi/debugging.md]: Done :)

File: content/hi/debugging.md [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+के रूप में कार्य नहीं करेंगे। कोई एप्लिकेशन कितना महत्वपूर्ण है, इस पर निर्भर करते हुए, बग का महत्वपूर्ण नकारात्मक प्रभाव हो सकता है - आर्थिक रूप से और मानव जीवन पर भी। 
+आमतौर पर, एप्लिकेशन कोड को विभिन्न चरणों या वातावरण से गुजरना पड़ता है जहां इसका परीक्षण किया जाता है। एक आवेदन जितना अधिक महत्वपूर्ण होगा, परीक्षण उतना ही सटीक होना चाहिए।
+
+## समाधान
+
+जब बग दिखाई देते हैं, तो इंजीनियर को उत्पादन प्रणालियों के लिए अवांछित व्यवहार को कम करने के लिए आवेदन को डिबग (जैसे, ढूंढना और ठीक करना) करना पड़ता है। 
+डिबगिंग कोई आसान काम नहीं है क्योंकि इंजीनियरों को अवांछित व्यवहार के स्रोत को ट्रैक करना होता है। इसे कोड के बारे में ज्ञान और रनटाइम पर निष्पादन संदर्भ की आवश्यकता होती है। 
+यह वह जगह है जहाँ विभिन्न डिबगिंग तकनीक और उपकरण काम में आते हैं। उदाहरण के लिए, लॉग, ट्रेस और मेट्रिक्स का विश्लेषण, उत्पादन में सीधे डिबगिंग के लिए उपयोग किया जाता है।
+संबंधित निष्पादन संदर्भ का विश्लेषण करते समय डेवलपर्स रनटाइम पर कोड के माध्यम से कदम उठाने के लिए इंटरैक्टिव डिबगिंग का उपयोग कर सकते हैं। एक बार जब वे विफलता के स्रोत की पहचान कर लेते हैं, 
+तो वे कोड को सही करते हैं और पैच बनाते हैं।
```
[the remark this answers is not in the document]
Comment (Omkar0114) [reply content/hi/debugging.md]: Done :)

File: content/hi/debugging.md [hunk 6]
```diff
@@ -0,0 +1,22 @@
...
+---
+
+## यह क्या है
+
+डिबगिंग यह एक वांछित परिणाम प्राप्त करने के लिए कंप्यूटर प्रोग्राम, सॉफ़्टवेयर या सिस्टम से बग को खोजने और हल करने की प्रक्रिया है। 
+बग एक दोष या समस्या है जिसके कारण गलत या अप्रत्याशित परिणाम मिलते हैं।
+
+## समस्या
+
+सॉफ्टवेयर विकास एक जटिल गतिविधि है जो बग को पेश किए बिना कोड लिखना लगभग असंभव बना देती है। इन बग के परिणामस्वरूप कोड होता है जो निष्पादित होने पर इच्छित (अपरिभाषित व्यवहार) के रूप में कार्य नहीं करेगा। कोई एप्लिकेशन कितना महत्वपूर्ण है, इस पर निर्भर करते हुए, बग का महत्वपूर्ण नकारात्मक प्रभाव हो सकता है - आर्थिक रूप से और मानव जीवन पर भी।
```
Comment (Garima-Negi) [inline content/hi/debugging.md:14]:
```suggestion
सॉफ्टवेयर विकास एक जटिल गतिविधि है जो बग को पेश किए बिना कोड लिखना लगभग असंभव बना देती है। इन बग के परिणामस्वरूप कोड होता है जो निष्पादित होने पर इच्छित (अपरिभाषित व्यवहार) रूप में कार्य नहीं करेगा। कोई एप्लिकेशन कितना महत्वपूर्ण है, इस पर निर्भर करते हुए, बग का महत्वपूर्ण नकारात्मक प्रभाव हो सकता है - आर्थिक रूप से और मानव जीवन पर भी। 
```

Comment (Omkar0114) [reply content/hi/debugging.md]: Updated :)

**Review by Garima-Negi — APPROVED** [review]

**Comment by Garima-Negi** [comment]
LGTM

**Comment by Omkar0114** [comment]
Thanks a lot. When will it be merged? 😃

**Comment by sayantani11** [comment]
LGTM

**Review by sayantani11 — APPROVED** [review]

**Comment by sayantani11** [comment]
@Omkar0114 Great work!
Thanks😃

**Comment by Omkar0114** [comment]
Thank You so much @Garima-Negi & @sayantani11 for merging my PR.
😃🤩Looking forward to contributing more! 👍

**Pull request merged**